content/blog/computer-ergonomics/index.md

```diff
@@ -0,0 +1,52 @@
+---
+title: Computer ergonomics
+date: "2019-06-03T12:12:03.284Z"
+description: Don't slouch.
+---
+
+> The definition of computer ergonomics is the practice of fitting the set up of the computer and the workspace to fit the user and the user's work needs in order to minimize physical stress on the computer user (Your dictionary).
+
+Developers sit in front of a computer for hours on end. As you may already know, sitting is not the most natural position for humans to be in for a prolonged period of time. Small repetitive strain on the body from bad posture can cause people to experience muscle strain and headaches.
+
+![Back pain](./back-pain.jpg)
+
+Being aware and mindful of the body position you are in during work can help reduce or completely mitigate bad posture habits. On the contrary, just getting a shiny new expensive standing desk will not solve all your problems.
+
+### Computer ergonomics
+
+The general principle of ergonomics is to keep the body in a neutral position and avoid slouching over. Take time to set up the work environment, so you can feel better and be more productive later.
+
+![Sit and stand posture](./sit-stand-posture.jpg)
+
+- **Computer screen**. Laptop screens are not ergonomically designed products, because the screen is too low and we lean in towards it. Having an external monitor about an arm's length away is the first step towards a better workplace. Eyes should be aligned about 5 cm (2 inches) below monitor height.
```

Comment:
i would change the order of these paragraphs a bit. 'computer screen' and 'chair' are okay, and then i would go with 'light', 'wrists' and 'breaks'